conda_smithy/templates/README.md.tmpl

```diff
@@ -16,40 +16,40 @@ Summary: {{ package.meta.about.summary }}
 Current build status
 ====================
 {%- set appveyor_url_name = github.repo_name.replace('_', '-').replace('.', '-') %}
-
-{%- set disabled_badge_commit = "90845bba35bec53edac7a16638aa4d77217a3713" %}
-{%- set disabled_badge_path = "conda_smithy/static/disabled.svg" %}
-{%- set disabled_badge_url = "https://cdn.rawgit.com/conda-forge/conda-smithy/%s/%s"|format(disabled_badge_commit, disabled_badge_path) %}
+{%- set shield = "https://img.shields.io/" %}
 {# #}
 
 {%- if noarch_python %}
 {%- if circle.enabled %}
-All platforms: [![Circle CI](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }}.svg?style=shield)](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }})
+All platforms:
+[![noarch]({{ shield }}circleci/project/github/{{ github.user_or_org }}/{{ github.repo_name }}.svg?label=noarch)](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }})
 {%- else %}
-All platforms: ![]({{ disabled_badge_url }})
+All platforms: ![noarch disabled]({{ shield }}badge/noarch-disabled-lightgrey.svg)
 {%- endif %}
 {%- else %}
 {%- if circle.enabled %}
-Linux: [![Circle CI](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }}.svg?style=shield)](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }})
+[![Linux]({{ shield }}circleci/project/github/{{ github.user_or_org }}/{{ github.repo_name }}.svg?label=linux)](https://circleci.com/gh/{{ github.user_or_org }}/{{ github.repo_name }})
 {%- else %}
-Linux: ![]({{ disabled_badge_url }})
+![Linux disabled]({{ shield }}badge/linux-disabled-lightgrey.svg)
 {%- endif %}
 {%- if travis.enabled %}
-OSX: [![TravisCI](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }}.svg?branch=master)](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }})
+[![OSX]({{ shield }}travis/{{ github.user_or_org }}/{{ github.repo_name }}.svg?label=OSX)](https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }})
 {%- else %}
-OSX: ![]({{ disabled_badge_url }})
+![Linux disabled]({{ shield }}badge/OSX-disabled-lightgrey.svg)
```

Comment:
Noticed a small typo here. Fixing in PR ( https://github.com/conda-forge/conda-smithy/pull/728 ).